doc/articles/uno-development/troubleshooting-memory-issues.md

```diff
@@ -101,3 +101,18 @@ grid.Children.Add(myItem);
 ```
 
 For Xamarin.iOS specifically, see [this article about performance tuning](https://learn.microsoft.com/xamarin/ios/deploy-test/performance).
+
+## Profiling on Desktop (Skia)
+
+On Desktop target, Visual Studio diagnostic session can be used to investigate application memory to find issues such as memory leaks. Follow [this guide from microsoft](https://learn.microsoft.com/en-us/visualstudio/profiling/memory-usage-without-debugging2?view=vs-2022&pivots=programming-language-dotnet).
+
+> [!TIP]
+> If the "Memory Usage" is not listed under the Available Tools of the Performance Profile. Make sure "YourProjectName (Desktop)" is selected as the target. If that still doesn't work, open the `YourProjectName.csproj` and manually replace the following line temporarily:
+> ```xml
+> <!--<TargetFrameworks>net9.0-android;net9.0-ios;net9.0-browserwasm;net9.0-desktop</TargetFrameworks>-->
+> <TargetFrameworks>net9.0-desktop</TargetFrameworks>
+> ```
+> You may need to close and re-open the solution after making the change.
+
+> [!TIP]
+> During profiling, you may use `GC.Collect()` to forces an immediate GC(Garbage Collection) without having to wait. It should be noted that the GC does not immediately take effect, and sometime it may take one more than one GC call for all loose object to be collected. For rigorousness, it recommended to GC twice and wait a few seconds before take a memory snapshot when troubleshooting memory leaks.
```

Comment:
```suggestion
> During profiling, you may use `GC.Collect()` to forces an immediate GC(Garbage Collection) without having to wait. It should be noted that the GC does not take effect immediately, and sometimes it may take more than one GC call for all loose objects to be collected. For rigorousness, it is recommended to call GC twice and wait a few seconds before taking a memory snapshot when troubleshooting memory leaks.
```

Reply:
@Xiaoy312  It was marked as resolved, but I still see some of the same typos in the text.